packages/app-content-pages/src/screens/About/components/OurMission.js

```diff
@@ -27,12 +65,7 @@ export default function OurMission() {
         <Trans
           i18nKey='AboutPage.ourMission.paragraphs.one'
           t={t}
-          components={[
-            <LinkInParagraph
-              key='AboutPage.ourMission.paragraphs.one'
-              href='/publications'
-            />
-          ]} // after switch to app-root, this will need to be /about/publications
+          components={[<Anchor key='publications-page' href='/publications' />]} // after switch to app-root, this will need to be /about/publications
```

Comment:
This component will eventually be a component library, lib-content-pages, so I'm removing Next Links for now.